src/Htmxor/Http/SwapStyleBuilder.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using System.Collections.Specialized;
+
+namespace Htmxor.Http;
+
+/// <summary>
+/// A builder class for constructing a swap style command string for HTMX responses.
+/// </summary>
+public class SwapStyleBuilder
+{
+    private readonly SwapStyle? style;
+    private readonly OrderedDictionary modifiers = new();
```

Comment:
Probably not, but I wanted to retain order because reusing the same modifier key should cause a modifier of the same name to be replaced.  e.g. You can't do ShowOnTop() and ShowOnBottom(), so whatever command that impacts the show modifier last is what is used.  If I used a Dictionary the act of removing keys can cause the dictionary to reorder, which I also don't want for testing purposes and consistency of output.